src/cogent3/core/new_alignment.py

```diff
@@ -3011,38 +3042,72 @@ class AlignedSeqsData(AlignedSeqsDataABC):
     def __init__(
         self,
         *,
-        seqs: Optional[dict[str, StrORBytesORArray]],
-        gaps: Optional[dict[str, numpy.ndarray]],
+        gapped_seqs: numpy.ndarray,
+        names: tuple[str],
         alphabet: new_alphabet.AlphabetABC,
-        offset: dict[str, int] = None,
+        ungapped_seqs: Optional[dict[str, numpy.ndarray]] = None,
+        gaps: Optional[dict[str, numpy.ndarray]] = None,
+        offset: Optional[DictStrInt] = None,
         align_len: OptInt = None,
         check: bool = True,
     ):
+        """
+        Parameters
+        ----------
+        gapped_seqs
+            2D numpy.uint8 array of aligned sequences. axis 0 are sequences,
+            axis 1 are alignment positions
+        names
+            sequence names in order matching the axis 0 of gapped_seqs
+        alphabet
+            caharacter alphabet for the sequences
+        ungapped_seqs
+            a dictionary mapping names to 1D numpy.uint8 arrays of individual
+            sequences without gaps. If not provided, computed on demand.
+        gaps, optional
```

Comment:
the comment that gaps is optional is not consistent with the rest of the docstring